frontend/src/i18n/locales/en.yaml

```
@@ -237,7 +243,13 @@ manage:
     no-path: Missing path! You opened an invalid link.
 
     general:
-      rename-label: Page name
+      page-name: Page name
+      name-directly: Name directly
+      name-directly-description: Set a fixed name for this page.
+      name-from-block: Derive name from video or series
+      name-from-block-description: >
+        Link a video/series-element from this page: the page name will always be the title of the linked element.
```

```suggestion
        Link a video/series-element from this page: The page name will always be the title of the linked element.
```

Full sentences after a colon should be capitalized.

---

We consistently use lowercase letters after a colon in English. Both is fine, with lowercase being more common when the second part is not a complete sentence. But ye, due to how we do it everywhere else, I would keep it as is.